x/wasm/keeper/bench_test.go

```diff
@@ -21,21 +48,21 @@ func BenchmarkExecution(b *testing.B) {
 			db:     func() dbm.DB { return dbm.NewMemDB() },
 			pinned: true,
 		},
-		"unpinned, level db": {
-			db: func() dbm.DB {
-				levelDB, err := dbm.NewGoLevelDBWithOpts("testing", b.TempDir(), &opt.Options{BlockCacher: opt.NoCacher})
-				require.NoError(b, err)
-				return levelDB
-			},
-		},
-		"pinned, level db": {
-			db: func() dbm.DB {
-				levelDB, err := dbm.NewGoLevelDBWithOpts("testing", b.TempDir(), &opt.Options{BlockCacher: opt.NoCacher})
-				require.NoError(b, err)
-				return levelDB
-			},
-			pinned: true,
-		},
+		//"unpinned, level db": {
```

Comment:
q: Why are the levelDB cases commented out? If they are not useful then let's drop them.